ci_framework/roles/reproducer/tasks/epel.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Enable CRB repositories (CentOS)
+  when:
+    - ansible_facts['distribution'] == 'CentOS'
+  become: true
+  ansible.builtin.command:
+    cmd: dnf config-manager --set-enabled crb
+
+- name: Enable CRB repositories (RHEL)
+  when:
+    - ansible_facts['distribution'] == 'RedHat'
+  become: true
+  vars:
+    arch: "{{ ansible_facts['architecture'] }}"
+  ansible.builtin.command:
+    cmd: >-
+      subscription-manager repos
+      --enable codeready-builder-for-rhel-9-{{ arch }}-rpms
+
+- name: Install EPEL repositories
+  become: true
+  vars:
+    rhel_uri: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm"
```

Comment:
Is it compatible with RHEL?
Would it be better to make it as a config option?

Reply:
that's the official RHEL way as documented by Red Hat.